spec/ttt_app/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../../lib/ttt_app.rb'
+
+# class TestTicTacToe < Test::Unit::TestCase
+#     def test_printWelcome
+#         assert_equal 'Welcome to Tic Tac Toe Game', TicTacToe.printWelcome, "TicTacToe.printWelcome should return a string  'Welcome to Tic Tac Toe Game'"
+#     end
+# end
+
+RSpec.describe TicTacToe do
+    it { expect(TicTacToe.printWelcome).to eql 'Welcome to Tic Tac Toe Game' }
+end
+
+RSpec.describe TicTacToe do
+    it { expect(TicTacToe.printWelcome).to eql 'Welcome to Tic Tac Toe Game' }
+end
```

Comment:
Any reason why there are two of the same test?

Reply:
It's an oversight. I'm correcting it right away